### PR TITLE
337709: Orphaned Containers Not Cleaned Up by Start Script for DBs 

### PR DESCRIPTION
**Pull request by santosh-gh**
[AB#337709](https://dev.azure.com/defragovuk/93c65135-d16b-49d6-a191-4a5313532779/_workitems/edit/337709)

# **What this PR does / why we need it**:
To reduce the number of re-tries if a job failed by setting backoffLimit in each of the Jobs.
The default backoffLimit  is 6, so if a job fails it retries another 6 more times hence create total 7 pods
By setting backoffLimit: 2 , it will create a max 3 pods if the job fails.

# **Special notes for your reviewer**
In each of the Job the backoffLimit  is set to 2.
so if the job fails, it will try another 2 times and totally max 3 pods will be created for the failed jobs.
If no backoffLimit is set by default it will create 7 pods for a failed job.

# Testing
*Any relevant testing information and pipeline runs.*
https://dev.azure.com/defragovuk/DEFRA-FFC/_build/results?buildId=536437&view=results

https://portal.azure.com/#@defradev.onmicrosoft.com/resource/subscriptions/55f3b8c6-6800-41c7-a40d-2adb5e4e1bd1/resourceGroups/SNDADPINFRG1402/providers/Microsoft.ContainerService/managedClusters/SNDADPINFAK1401/workloads

# Checklist (please delete before completing or setting auto-complete)
- [x] Story Work items associated (not Tasks)
- [x] Successful testing run(s) link provided
- [x] Title pattern should be `{work item number}: {title}`
- [x] Description covers all the changes in the PR
- [ ] This PR contains documentation
- [ ] This PR contains tests

# **How does this PR make you feel**:

### CODE REVIEW

File: services/environments/snd/base/image-update.yaml [hunk 1]
```diff
@@ -12,14 +12,14 @@ spec:
   git:
     checkout:
       ref:
-        branch: main
+        branch: sm/337709/OrphanedContainers
```
Comment (kbabigumira) [inline services/environments/snd/base/image-update.yaml:15]: Revert back to main branch

File: services/environments/snd/base/image-update.yaml [hunk 2]
```diff
@@ -12,14 +12,14 @@ spec:
...
     commit:
       author:
         email: fluxcdbot@users.noreply.github.com
         name: fluxcdbot
       messageTemplate: '{{range .Updated.Images}}{{println .}}{{end}}'
     push:
-      branch: main
+      branch: sm/337709/OrphanedContainers
```
Comment (kbabigumira) [inline services/environments/snd/base/image-update.yaml:22]: revert back to main branch

File: services/ffc/ffc-ffd/ffc-ffd-backend-poc/pre-deploy/migration/snd/01/migration-patch.yaml [hunk 3]
```diff
@@ -15,7 +15,7 @@ spec:
         - name: SCHEMA_NAME
           value: ${POSTGRES_SCHEMA_NAME}
         - name: POSTGRES_DB
-          value: ${POSTGRES_DB}
+          value: ${POSTGRES}
```
Comment (kbabigumira) [inline services/ffc/ffc-ffd/ffc-ffd-backend-poc/pre-deploy/migration/snd/01/migration-patch.yaml:18]: Why was this changed?

**Review by kbabigumira — CHANGES_REQUESTED** [review]

File: services/environments/snd/base/image-update.yaml (hunk 1 above)
Comment (santosh-gh) [reply services/environments/snd/base/image-update.yaml]: Reverted to main

File: services/ffc/ffc-ffd/ffc-ffd-backend-poc/pre-deploy/migration/snd/01/migration-patch.yaml (hunk 3 above)
Comment (santosh-gh) [reply services/ffc/ffc-ffd/ffc-ffd-backend-poc/pre-deploy/migration/snd/01/migration-patch.yaml]: Reverted to main

File: services/environments/snd/base/image-update.yaml (hunk 2 above)
Comment (santosh-gh) [reply services/environments/snd/base/image-update.yaml]: reverted to main branch

**Review by kbabigumira — APPROVED** [review]

**Review by ppotturi — APPROVED** [review]

**Pull request merged**